Student/StudentE2ETests/Announcements/AnnouncementsTests.swift

```diff
@@ -23,6 +23,14 @@ class AnnouncementsTests: E2ETestCase {
     typealias DetailsHelper = Helper.Details
     typealias AccountNotifications = Helper.AccountNotifications
 
+    override func tearDown() {
+        super.tearDown()
+
+        // Disable discussion redesign feature flag
+        let featureFlagResponse = seeder.setFeatureFlag(featureFlag: .newDiscussion, state: .off)
+        XCTAssertEqual(featureFlagResponse.state, DSFeatureFlagState.off.rawValue)
```

Comment:
`super.tearDown()` should be the last to call in this override.

Reply:
Correct, done.